### ISSUE TITLE
Feedback: long time of building

### ISSUE THREAD

**Issue by AugustinMauroy** [JamesSingleton/redshirt-sports#66]
A point which unnecessarily slows down the build time of this [page](https://github.com/JamesSingleton/redshirt-sports/blob/main/app/(website)/%5Bslug%5D/page.tsx). You currently deploy your site on Vercel. But this page is built in static. So you're not using the power of Vercel and NextJS at all. To make this page dynamic, just remove the `generateStaticParams`.

**Comment by JamesSingleton** [comment]
I don't think that is the issue as the Node.js website has https://github.com/nodejs/nodejs.org/blob/main/app/%5Blocale%5D/%5B%5B...path%5D%5D/page.tsx#L38.

Ideally they should be statically built for SEO purposes shouldn't it?
